xbmc/cores/RetroPlayer/rendering/RPRenderManager.cpp

```diff
@@ -74,6 +80,13 @@ void CRPRenderManager::Deinitialize()
     buffer->Release();
   m_pendingBuffers.clear();
 
+  for (auto it : m_savestateBuffers)
+  {
+    for (auto renderBuffer : it.second)
```

Comment:
```suggestion
    for (auto& renderBuffer : it.second)
```

Reply:
`renderBuffer` is of type `IRenderBuffer*` so it doesn't need to assign by reference, correct?